bags:
  tartandriver:
    storage_config: mcap_qos.yaml
    args:
      -s: mcap
      -d: 120
    hooks:
      pre_logger:
        - routines.example_routine.pre_logger
      post_logger:
        - routines.example_routine.post_logger
        - routines.freq.bag_freq
    topics:
      - $ALL_TOPICS
      # - $ACTUATION_TOPICS
      # - $NOVATEL_TOPICS
      # - $MULTISENSE_TOPICS
      # - $VELODYNE_TOPICS
      # - $CORE_TOPICS
      # - $SENSOR_TOPICS
      # - $THERMAL_TOPICS
      # - $AUTO_TOPICS

